tests/server/statsd-collector.js

```diff
@@ -76,6 +76,34 @@ define([
 
   };
 
+  suite['properly collects metrics events with ab testing tags'] = function () {
+    var dfd = new Promise.Deferred();
+
+    var metricsCollector = new StatsDCollector();
+    metricsCollector.init();
+
+    var fixtureMessage = JSON.parse(fs.readFileSync('tests/server/fixtures/statsd_body_ab.json'));
```

Comment:
This is inside of the AMD cotext, therefore `require` is AMD's require. so we cannot use common js here, unless something like `require('intern/dojo/node!../path/to/file')`. So it is possible, but not easier